### ISSUE TITLE
Problem adding Plex users

### ISSUE THREAD

**Issue by joelstitch** [itsToggle/plex_debrid#642]
I got Plex debrid running great on my server but I am having issues adding other Plex users. On my user I am able to add things to my watch list and have them doenloaded. On the Plex debrid script I add the new user and the token (which I am guessing is my Plex server token?) but when they add things to their watch list it doesn't download. If there is a specific token for each user then I don't know how to find it because when they try to see the xml file to see the token it doesn't even gets them the option on Plex.

**Comment by truncsphere** [comment]
I seem to be having an issue still adding shared user(s). Tried via username (and email) and their accesstoken and I get a 401 not authorized. 
Tried their username and admin token, nothing; username and server token still nothing. 

Any ideas?

**Comment by joelstitch** [comment]
I actually haven't gotten it to work. I thought I did but it only works with my username, I did added other people username but nothing they add to their watchlist gets added.

**Comment by truncsphere** [comment]
That is unfortunate. Not sure what other combo to try at this point.

**Comment by hernandito** [comment]
It would be great if this could get resolved. It would be great if my Plex friends could add items.

**Comment by joelstitch** [comment]
I taught I got it to work but it was actually OMBI downloading the movies requested on plex watchlist. Now I turned off OMBI and still can't get this to work with other plex users.

**Comment by truncsphere** [comment]
So apparently the token being used by PD is different then token we get for users using the normal method. I used the method below and it worked. 
This is the method as described by zurg. 
`open Plex in a browser, open dev console and copy-paste this: window.localStorage.getItem("myPlexAccessToken")`

**Comment by joelstitch** [comment]
> So apparently the token being used by PD is different then token we get for users using the normal method. I used the method below and it worked. This is the method as described by zurg. `open Plex in a browser, open dev console and copy-paste this: window.localStorage.getItem("myPlexAccessToken")`

Does that gives you a second user token?

**Comment by hernandito** [comment]
The above worked my friend!!! 

Only drawback is that I need to know each of my user names / passwords to get this token. I have to be logged as that user. The dev console instructions are not easy to follow for my non-techie friends and family. But that there is a solution is obviously great!

Thank you @truncsphere.

**Comment by hernandito** [comment]
> > So apparently the token being used by PD is different then token we get for users using the normal method. I used the method below and it worked. This is the method as described by zurg. `open Plex in a browser, open dev console and copy-paste this: window.localStorage.getItem("myPlexAccessToken")`
> 
> Does that gives you a second user token?

Yes... a non managed user that is sharing my libraries.

**Comment by joelstitch** [comment]
> The above worked my friend!!!
> 
> Only drawback is that I need to know each of my user names / passwords to get this token. I have to be logged as that user. The dev console instructions are not easy to follow for my non-techie friends and family. But that there is a solution is obviously great!
> 
> Thank you @truncsphere.

Yeah would you explain the Dev menu thing. I just tried on Chrome and theres so much stuff that pops up on Dev menu that I had no idea where im supposed to put the text.

**Comment by truncsphere** [comment]
> > The above worked my friend!!!
> > Only drawback is that I need to know each of my user names / passwords to get this token. I have to be logged as that user. The dev console instructions are not easy to follow for my non-techie friends and family. But that there is a solution is obviously great!
> > Thank you @truncsphere.
> 
> Yeah would you explain the Dev menu thing. I just tried on Chrome and theres so much stuff that pops up on Dev menu that I had no idea where im supposed to put the text.

I use FF myself and F12 opens the console, the cursor is already in the console where you can paste the above command in. Once you hit enter it will return the token for the logged in user.

**Comment by hernandito** [comment]
> > > The above worked my friend!!!
> > > Only drawback is that I need to know each of my user names / passwords to get this token. I have to be logged as that user. The dev console instructions are not easy to follow for my non-techie friends and family. But that there is a solution is obviously great!
> > > Thank you @truncsphere.
> > 
> > 
> > Yeah would you explain the Dev menu thing. I just tried on Chrome and theres so much stuff that pops up on Dev menu that I had no idea where im supposed to put the text.
> 
> I use FF myself and F12 opens the console, the cursor is already in the console where you can paste the above command in. Once you hit enter it will return the token for the logged in user.

In Firefox, I got a warning that that pasting wasn't allowed in the console... that complicates things.... And believe it or not, my sister doesn't have a computer. She does everything on her phone.

**Comment by hernandito** [comment]
It also works w/ managed users within my account.

**Comment by joelstitch** [comment]
> It also works w/ managed users within my account.

Without needing their token?

**Comment by hernandito** [comment]
> > It also works w/ managed users within my account.
> 
> Without needing their token?

No... you still need to be logged in as the user and go into dev console.
